packages/stem_flutter/analysis_options.yaml

```
@@ -0,0 +1,7 @@
+include: package:very_good_analysis/analysis_options.yaml
+
+analyzer:
+  exclude:
+    - "**/*.g.dart"
+    - "**/*.orm.dart"
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)